AtomicTestHarnesses.psd1

```diff
@@ -64,6 +66,12 @@ PrivateData = @{
         # ReleaseNotes of this module
         ReleaseNotes = @'
 
+2.0.0

```

Comment:
Version in this case should be 1.10.0 and ModuleVersion should be rev'd to 1.10.0.0 accordingly